### DISCUSSION TITLE
Add ever-solidity in codewars

### DISCUSSION THREAD

**Discussion by kobzistiy** [everscale-org/bounties#23] [Ideas]
Add ever-solidity in codewars https://www.codewars.com/. 
Git: https://github.com/codewars/codewars.com

Add Сode runner: https://github.com/codewars/runner
Add Documentation: https://github.com/codewars/docs

**Comment by ilyar** [comment]
I think it would be rational to solve this https://github.com/everscale-org/bounties/discussions/22 first of all. it’s more clear how to add it, and also there you will also have to make a runner, which I think will be reused
